sdk/search/search-documents/swagger/Data.md

```diff
@@ -55,28 +58,5 @@ directive:
   - from: swagger-document
     where: $.definitions.IndexAction
     transform: >
-      $.properties['@search.action']['x-ms-client-name'] = '__actionType';
       $.required = ['@search.action'];
 ```
-
-
-### Change text to _text in SuggestResult
-
-```yaml
-directive:
-  - from: swagger-document
-    where: $.definitions.SuggestResult.properties['@search.text']
```

Comment:
It seems like we're relying on a lack of name collision for the common property name of `text` now, which feels pretty risky.

Reply:
Fixed now.